appcelerator-archive/amp
# Minor Updates to cloudformation template

Minor updates to cloudformation stack template `bootstrap.yml` due to the changes to the bootstrap script.

to test:
 - Log on to AWS and go to Cloudformation
 - Create a new stack and choose `bootstrap.yml` in the "Upload a template to Amazon S3" option.
 - Give the stack a name and add an ssh key pair to the cluster properties.
 - Finish creating the stack and ensure that the following EC2 instances are created.
 - 1 controller instance. 
 - 3 manager instances.
 - 2 worker instances.
